.rubocop.yml

```diff
@@ -27,6 +27,10 @@ Layout:
 Lint:
   Enabled: true
 
+Lint/MissingSuper:
+  Exclude:
+    - "app/components/*"
```

Comment:
por qué deshabilitar esto?

Reply:
Lo expliqué en el commit 🤓:

> Rubocop is complaining about components not calling `super` in their
> initializer:
> 
> ```
> app/components/approvable_checkbox_component.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
>   def initialize(approvable:, subject_show:, current_student:) ...
>   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> app/components/checkbox_input_component.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
>   def initialize(*args) ...
>   ^^^^^^^^^^^^^^^^^^^^^
> ```
> 
> This is how components are used so let's exclude the `Lint/MissingSuper`
> Cop for running in the `app/components` folder.

Reply:
no sé si entiendo a qué te referís con "This is how components are used"

No deberíamos llamar a `super` por las dudas?

Reply:
Me refería a que [en la documentación de `view_component`](https://viewcomponent.org/) e incluso cuando generás un componente usando los generators de `view_component`, nunca llaman a `super`.

Supongo que podríamos llamarlo por las dudas, pero entiendo que no es como recomiendan usarlo así que no sé del todo qué podría pasar.

Reply:
https://github.com/rubocop/ruby-style-guide/issues/809#issuecomment-671688715

Reply:
Me imagino que eso es porque el initializer de `ViewComponent::Base` está vacío, pero también supongo que la regla de rubocop es más por una buena práctica de Ruby de siempre llamar a `super` en el initializer de una subclase, para asegurarte que llamas al initializer del padre

Reply:
Si, entiendo. Yo, personalmente, siento que si no es necesario no lo haría pero no tengo una "strong opinion" 🙂

Reply:
A mi me resulta extraño deshabilitarlo solo para los componentes.
Si no hay una buena razón, preferiría dejarlo habilitado. Pero si lo deshabilitamos, me parece que debería ser para todos los archivos.